test/test_sarif.ml

```diff
@@ -1,139 +1,144 @@
-open Core
 open Sarif
 
+let read_all file =
+  let ic = open_in_bin file in
+  let s = really_input_string ic (in_channel_length ic) in
+  close_in ic;
+  s
```

Comment:
This fn is a duplicate in the malformed.ml, we can perhaps put it in a util.ml file. But for testing this doesnt really matter meh.....